src/components/table/AposTable.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <table class="apos-table">
+    <caption
+      class="apos-table__caption apos-sr-only"
+      role="alert"
+      aria-live="polite"
+    >
+      {{
+        data.caption
+      }}
+    </caption>
+    <slot name="tableHeader">
+      <AposTableHeader :columns="data.header" :selectable="selectable" />
+    </slot>
+    <tbody class="apos-table__body" v-if="rows.length">
+      <slot name="tableRow">
+        <AposTableRow
+          v-for="(row, index) in rows"
+          :key="index"
+          :row="row"
+          :id="index"
+          :selectable="selectable"
+        />
+      </slot>
+    </tbody>
+    <tbody v-else class="apos-table__body">
+      <tr class="apos-table__row">
+        <td class="apos-table__cell-empty" :colspan="data.header.length">
+          Nothing to see here... Move along
```

Comment:
we have an empty state component we can use here, `AposEmptyState`

Reply:
This won't be needed here. It'll be handled a level up.